docs/howto/openstack/magnum/new-k8s-cluster.md

```diff
@@ -71,82 +70,99 @@ package manager of your operating system or `pip`:
 
     You can now list all available templates in the region:
 
-    ```bash
-    openstack coe cluster template list
-    ```
+    ```console
+    $ openstack coe cluster template list
 
-    ```plain
-    +--------------------------------------+----------------------------------------------------------------+------+
-    | uuid                                 | name                                                           | tags |
-    +--------------------------------------+----------------------------------------------------------------+------+
-    | 3f476f01-b3de-4687-a188-6829ed947db0 | Kubernetes 1.15.5 on Fedora-atomic 29 4C-8GB-20GB No Master LB | None |
-    | c458f02d-54b0-4ef8-abbc-e1c25b61165a | Kubernetes 1.15.5 on Fedora-atomic 29 2C-4GB-20GB No Master LB | None |
-    | f9e1a2ea-b1ff-43e7-8d1e-6dd5861b82cf | Kubernetes 1.18.6 on Fedora-coreos 33 2C-4GB-20GB No Master LB | None |
-    +--------------------------------------+----------------------------------------------------------------+------+
+    +--------------------------------------+---------------------------------------------+------+
+    | uuid                                 | name                                        | tags |
+    +--------------------------------------+---------------------------------------------+------+
+    | 3f476f01-b3de-4687-a188-6829ed947db0 | Kubernetes 1.15.5 on Fedora-atomic 29       | None |
+    |                                      | 4C-8GB-20GB No Master LB                    |      |
+    | c458f02d-54b0-4ef8-abbc-e1c25b61165a | Kubernetes 1.15.5 on Fedora-atomic 29       | None |
+    |                                      | 2C-4GB-20GB No Master LB                    |      |
+    | f9e1a2ea-b1ff-43e7-8d1e-6dd5861b82cf | Kubernetes 1.18.6 on Fedora-coreos 33       | None |
+    |                                      | 2C-4GB-20GB No Master LB                    |      |
+    | 59bd894b-0f5f-4a6e-98d3-a3eb7040faab | Kubernetes v1.23.3 on Fedora-coreos 35      | None |
+    | 9ca03308-996e-4eaa-b507-5730dcc19fcc | Kubernetes v1.24.16 on Fedora-coreos 37     | None |
+    +--------------------------------------+---------------------------------------------+------+
     ```
 
     Select the template you want by setting the corresponding `uuid`
     value to the  `CLUSTER_TMPL` variable:
 
-    ```bash
-    CLUSTER_TMPL="f9e1a2ea-b1ff-43e7-8d1e-6dd5861b82cf" # just an example
+    ```console
+    $ CLUSTER_TMPL="9ca03308-996e-4eaa-b507-5730dcc19fcc" # just an example
     ```
 
     Then, list all available keypairs...
 
-    ```bash
-    openstack keypair list
-    ```
+    ```console
+    $ openstack keypair list
 
```

Comment:
I believe we can drop the extraneous blank line here.

Reply:
Well, I know those are not really there, but I usually add them for improved readability. On the other hand, we can definitely **not** add those extra blank lines, and no one will complain.